### PR TITLE
Added info on RTM agent/proxy and link to troubleshooting

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by rtom-mulesoft-zz — DISMISSED** [review]

**Review by aljrubior — APPROVED** [review]
LGTM

File: modules/ROOT/pages/rtm-agent-proxy-config.adoc [hunk 1]
```diff
@@ -57,6 +57,28 @@ This example configures the agent to work with a proxy server that requires auth
 amc_setup -H myToken myMuleServer -P acme.proxy.com 443 internalAdmin Ins1d3V0icePassword
 ----
 
+== Verify That the Proxy Server Does Not Modify the Agent Certificate
```
Comment (aljrubior) [inline modules/ROOT/pages/rtm-agent-proxy-config.adoc]: Verify That the Proxy Server Does Not Modify the **Runtime Manager Certificate**

**Review by aljrubior — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/rtm-agent-proxy-config.adoc [hunk 2]
```diff
@@ -57,6 +57,28 @@ This example configures the agent to work with a proxy server that requires auth
...
 ----
 
+== Verify That the Proxy Server Does Not Modify the Agent Certificate 
+
+To ensure that your firewall or proxy does not intercept or modify the 
+Runtime Manager agent, run the following command:
+
+[source,console,linenums]
+----
+echo -e "GET / HTTP/1.0\r\n" | openssl s_client -connect mule-manager.anypoint.mulesoft.com:443 -ign_eof
```
Comment (aljrubior) [inline modules/ROOT/pages/rtm-agent-proxy-config.adoc:89]: Taking into consideration that 
- Old agents use `mule-manager.anypoint.mulesoft.com`
- Newer agents use `runtime-manager.anypoint.mulesoft.com` (+1.12.0 and +2.2.0) 

I think we better use `runtime-manager.anypoint.mulesoft.com` for reference.

Run the command with either host will probe if the firewall or proxy is tampering the certificate

File: modules/ROOT/pages/rtm-agent-proxy-config.adoc [hunk 3]
```diff
@@ -57,6 +57,28 @@ This example configures the agent to work with a proxy server that requires auth
...
+----
+
+The output of this command should include the following information:
+
+[source,console,linenums]
+----
+-----END CERTIFICATE-----
+subject=/C=US/O=Hybrid/OU=MuleSoft/CN=mule-manager.anypoint.mulesoft.com
```
Comment (aljrubior) [inline modules/ROOT/pages/rtm-agent-proxy-config.adoc:97]: For `runtime-manager.anypoint.mulesoft.com`

the output would be

`
-----END CERTIFICATE-----
subject=/C=US/ST=ca/L=San Francisco/O=MuleSoft, LLC/OU=Mulesoft/CN=runtime-manager.anypoint.mulesoft.com
issuer=/C=US/O=DigiCert Inc/CN=DigiCert SHA2 Secure Server CA
---`

**Review by aljrubior — APPROVED** [review]

**Review by aljrubior — CHANGES_REQUESTED** [review]
Leave some comments

**Review by aljrubior — APPROVED** [review]
LGTM

**Review by kbostrom — APPROVED** [review]
Edited per scope of request: no comments

**Pull request merged**